Hrychyna.University.InteractivePlantCareAssistant.Web/script.js

```diff
@@ -0,0 +1,133 @@
+let currentThread = null;
+
+function addThread() {
+    var input = document.getElementById('newThreadTitle');
+    var threadTitle = input.value.trim();
+    if (threadTitle === "") {
+        alert("Please enter a topic name.");
+        return;
+    }
+
+    var threadsList = document.getElementById('threadsList');
+    var li = document.createElement('li');
+
+    var userName = "Dariia";
+    var currentTime = new Date().toLocaleString("uk-UA");
+
+    li.innerHTML = `<span class="thread-title">${threadTitle}</span> <span class="thread-info">(Created by ${userName} at ${currentTime})</span>
```

Comment:
Use template literals for cleaner and more readable HTML string concatenation.

Reply:
Thanks for the comment. I'll leave things as they are for now, but I'll consider this suggestion later.